src/NuGetForUnity/Editor/Ui/NugetWindow.cs

```diff
@@ -711,88 +594,88 @@ private void DrawPackagesSplittedByManuallyInstalled(List<INugetPackage> package
         private void DrawOnline()
         {
             DrawOnlineHeader();
-            var headerStyle = GetHeaderStyle();
-
-            if (selectedPackageInstalls.Count > 0)
-            {
-                DrawSelectedForInstallationHeader(headerStyle);
-            }
 
             // display all of the packages
             scrollPosition = EditorGUILayout.BeginScrollView(scrollPosition);
             EditorGUILayout.BeginVertical();
 
-            IEnumerable<INugetPackage> packagesToShow;
-
-            if (selectedPackageInstalls.Count > 0 && showPackagesToInstall)
-            {
-                packagesToShow = availablePackages.Where(p => selectedPackageInstalls.Contains(p));
-            }
-            else
+            if (selectedPackageInstalls.Count > 0)
             {
-                packagesToShow = availablePackages.Where(p => !selectedPackageInstalls.Contains(p));
+                DrawSelectedForInstallationHeader();
+                if (showPackagesToInstall)
+                {
+                    DrawPackages(availablePackages.Where(p => selectedPackageInstalls.Contains(p)), true);
+                }
             }
 
-            DrawPackages(packagesToShow, true);
-
             // If user deselected all the packages revert to showing available packages
             if (selectedPackageInstalls.Count == 0)
             {
-                showPackagesToInstall = false;
+                showOnlinePackages = true;
             }
 
-            var showMoreStyle = GetHeaderStyle();
-            EditorGUILayout.BeginVertical(showMoreStyle);
+            if (selectedPackageInstalls.Count > 0)
+            {
+                var foldoutRect = EditorGUILayout.GetControlRect(false, 22f);
+                EditorGUI.DrawRect(foldoutRect.Expand(2f), Styles.FoldoutHeaderColor);
+                EditorGUI.DrawRect(foldoutRect.ExpandX(2f).AddY(-2f).SetHeight(1f), Styles.LineColor);
+                EditorGUI.DrawRect(foldoutRect.ExpandX(2f).AddY(foldoutRect.height + 1f).SetHeight(1f), Styles.LineColor);
+                showOnlinePackages = EditorGUI.Foldout(foldoutRect, showOnlinePackages, "Online packages", true);
+            }
 
-            if (showPackagesToInstall)
+            if (showOnlinePackages)
             {
-                var arrow = !showPackagesToInstall ? ArrowTipUp : ArrowTipDown;
-                if (GUILayout.Button($" {arrow} Online packages", headerStyle, GUILayout.Height(25f)))
-                {
-                    showPackagesToInstall = !showPackagesToInstall;
-                }
+                DrawPackages(availablePackages.Where(p => !selectedPackageInstalls.Contains(p)), true);
             }
 
+            EditorGUILayout.BeginVertical();
```

Comment:
Why do you need this BeginVertical? Aren't we already in vertical scope here?

Reply:
This was unnecessary, so I removed it.